arch/arm/soc/atmel_sam/same70/soc.h

```diff
@@ -48,6 +48,51 @@
 
 #endif /* _ASMLANGUAGE */
 
+/** Peripheral Hardware Request Line Identifier */
+#define DMA_PERID_HSMCI_TX_RX 0
```

Comment:
are these id's unique from peripheral-id we have in the dts for each device node?